vignettes/get_started.Rmd

```diff
@@ -0,0 +1,101 @@
+---
+title: "Get Started"
+output: 
+  rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Get Started}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+# Purpose of admiralci
+
+This repository contains GitHub Actions continuous integration/continuous delivery (CI/CD) workflows, most of which are used by [`Admiral`](https://github.com/pharmaverse/admiral) and its extensions.
+Workflows defined here are responsible for assuring high package quality standards without compromising performance, security, or reproducibility.
+
+Please refer to the [`.github/workflows`](.github/workflows) directory to view the source code for the GitHub Actions workflows.
+
+Notes :
+
+- Some workflows are using github actions from [InsightEngineering](https://github.com/insightsengineering/) group.
+<!-- - Most of the workflows tends to be use by other packages (`oak` package for example). A global repository at Pharmaverse organization level will be set-up to host all common workflows for pharmaverse packages (Beyond Admiral uses cases).
+This package will then probably be replaced by this global Pharmaverse repository in the future. -->
+- Most of the workflows are sharing the same steps (see [Common workflows structure](common_structure.html)) (boilerplate code)
+
+
+# Available workflows
+
+## Workflows trigger by Admiral MR (feature branch to main branch)
```

Comment:
Typo: "trigger" instead of "triggered" (across other headings too)

Reply:
```suggestion
## Workflows triggered by Admiral MR (feature branch to main branch)
```